install.sh

```diff
@@ -429,6 +429,20 @@ EOF
 	fi
 fi
 
+# Need to enable calico before starting rootlesskit
+if [ "$cni" = "calico" ]; then
+	cat <<EOF >>${config_dir}/usernetes/env
+U7S_CALICO=1
+EOF
+fi
+
+### Secret encryption
+if [ ! -f ${config_dir}/usernetes/master/secrets-encryption.yml ]; then
+	INFO "Enabling secrets encryption"
+	cp ${base}/manifests/secrets-encryption.yml ${config_dir}/usernetes/master/secrets-encryption.yml
+	echo "$(cat /dev/urandom | head -c 32 | base64 -w 0)" >> ${config_dir}/usernetes/master/secrets-encryption.yml
```

Comment:
```suggestion
	echo "$(cat /dev/urandom | head -c 32 | base64 -w 0)" > ${config_dir}/usernetes/master/secrets-encryption.yml
```

Reply:
Wouldn't that replace the whole config file instead of appending the key ?

Reply:
Sorry, misunderstood the script